server/app/modules/DurableJobModule.java

```diff
@@ -86,8 +86,10 @@ public DurableJobRegistry provideDurableJobRegistry(
     durableJobRegistry.register(
         DurableJobName.REPORTING_DASHBOARD_MONTHLY_REFRESH,
         persistedDurableJob ->
-            new ReportingDashboardMonthlyRefreshJob(reportingRepository, persistedDurableJob),
-        new RecurringJobExecutionTimeResolvers.FirstOfMonth2Am());
+            new ReportingDashboardMonthlyRefreshJob(
+                reportingRepositoryFactory.create(), persistedDurableJob),
+        //        new RecurringJobExecutionTimeResolvers.FirstOfMonth2Am());
```

Comment:
**nitpick:** Remove commented code

Reply:
Sure thing. I added it for testing locally. Actually I will remove the "immediately" resolver.